parent/pom.xml

```
@@ -138,16 +121,9 @@
                     </configuration>
                 </plugin>
                 <plugin>
-                    <groupId>org.sonatype.plugins</groupId>
-                    <artifactId>nexus-staging-maven-plugin</artifactId>
-                    <version>1.6.14</version>
-                    <configuration>
-                        <serverId>ossrh</serverId>
-                        <nexusUrl>${sonatypeOssDistMgmtNexusUrl}</nexusUrl>
-                        <autoReleaseAfterClose>false</autoReleaseAfterClose>
-                        <!-- Skip based on the maven.deploy.skip property -->
-                        <skipNexusStagingDeployMojo>${maven.deploy.skip}</skipNexusStagingDeployMojo>
-                    </configuration>
+                    <groupId>org.sonatype.central</groupId>
+                    <artifactId>central-publishing-maven-plugin</artifactId>
+                    <version>0.7.0</version>
```

```suggestion
                    <version>0.8.0</version>
```
suggestion: a new version has some out since this was originally created.